python/3d/auto_generated/symbols/qgspointcloud3dsymbol.sip.in

```diff
@@ -155,6 +155,26 @@ than this threshold value will not be rendered, see :py:func:`~QgsPointCloud3DSy
     virtual void copyBaseSettings( QgsAbstract3DSymbol *destination ) const;
 
 
+    virtual bool updateCurrentSymbol( QgsPointCloud3DSymbol *symbol );
+%Docstring
+Updates the current point cloud symbol to reflect the rendering settings as ``symbol``
+Returns true if the scene needs to be reloaded to reflect changes from ``symbol``
+
+.. versionadded:: 3.28
+%End
+  signals:
+
+    void pointSizeChanged( float newSize );
+%Docstring
+Emitted when the point size has changed.
+
+.. note::
+
+   This signal is only used to manage updating internal Qt3D QParameter object without reloading the whole scene
```

Comment:
EIther this is a useful signal in general and should be exposed to Python (in which case the note isn't needed), or it's for internal use only and should be hidden from sip